pages/items/[id].jsx

```diff
@@ -0,0 +1,9 @@
+import DetailsCard from "../../components/details/DetailsCard";
+
+export default function DetailsPage({ itemData }) {
+  return (
+    <div>
+      <DetailsCard items={itemData.wishlistItems} />
+    </div>
```

Comment:
See my comment above regarding `itemData`.
Additionally, there is a `div` here, you do not need.
```suggestion
      <DetailsCard items={itemData.wishlistItems} />
```